From 6c74a9c18bd3abfcd8167e16d22260be9f5e59c4 Mon Sep 17 00:00:00 2001
From: "yang.wang" <yang.wang@deepcomputing.io>
Date: Thu, 13 Mar 2025 17:31:32 +0800
Subject: [PATCH 229/534] dts: eswin-win2030-die1-soc.dtsi: Fix compile
 warnings.

arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5: Warning (spi_bus_bridge): /soc/spi1@70810000: node name for SPI buses should be 'spi'
  also defined at arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5
  also defined at arch/riscv/boot/dts/eswin/eic7702-evb-a1.dts:1092.10-1114.3
arch/riscv/boot/dts/eswin/eic7702-evb-a1.dtb: Warning (spi_bus_reg): Failed prerequisite 'spi_bus_bridge'
arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5: Warning (spi_bus_bridge): /soc/spi1@70810000: node name for SPI buses should be 'spi'
  also defined at arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5
  also defined at arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dts:1132.10-1154.3
arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dtb: Warning (spi_bus_reg): Failed prerequisite 'spi_bus_bridge'
arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5: Warning (spi_bus_bridge): /soc/spi1@70810000: node name for SPI buses should be 'spi'
  also defined at arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5
  also defined at arch/riscv/boot/dts/eswin/eic7702-tb.dts:1145.10-1167.3
arch/riscv/boot/dts/eswin/eic7702-tb.dtb: Warning (spi_bus_reg): Failed prerequisite 'spi_bus_bridge'
arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5: Warning (spi_bus_bridge): /soc/spi1@70810000: node name for SPI buses should be 'spi'
  also defined at arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi:1678.26-1693.5
  also defined at arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03.dts:963.10-985.3
arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03.dtb: Warning (spi_bus_reg): Failed prerequisite 'spi_bus_bridge'

Signed-off-by: yang.wang <yang.wang@deepcomputing.io>
---
 arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi b/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi
index 309c9ee991d2..cedb2490af55 100644
--- a/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi
+++ b/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi
@@ -1675,7 +1675,7 @@ d1_sdio1: mmc@0x70470000{
 			dma-noncoherent;
 		};
 
-		d1_ssi0: spi1@70810000 {
+		d1_ssi0: spi@70810000 {
 			compatible = "snps,eic770x-spi";
 			#address-cells = <1>;
 			#size-cells = <0>;
-- 
2.53.0

